iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -593,7 +593,7 @@ private Path getColStatsPath(Table table) {
   }
 
   private Path getColStatsPath(Table table, long snapshotId) {
-    return new Path(table.location() + STATS + table.name() + snapshotId);
+    return new Path(table.location() + STATS + snapshotId);
```

Comment:
would be nice to have some Unit tests that creates and checks the col stats file

Reply:
Added test https://github.com/apache/hive/pull/4871/commits/1d78a39c56a7fca223447b14033eb277d8dc9274

Reply:
thanks @zhangbutao